extensions/cornerstone/src/services/ViewportActionCornersService/ViewportActionCornersService.ts

```diff
@@ -0,0 +1,45 @@
+import { PubSubService } from '@ohif/core';
+
+export const VIEWPORT_ACTION_ARROWS_COMPONENT_ID = 'viewportActionArrowsComponent';
+export const VIEWPORT_STATUS_COMPONENT_ID = 'viewportStatusComponent';
```

Comment:
not sure what values these add, I see you import them in the provider, why not put it there?